### DISCUSSION TITLE
Error could not evaluate key path

### DISCUSSION THREAD

**Discussion by leorkhx** [Yue-Yang/ChatGPT-Siri#86] [Q&A]
![image](https://user-images.githubusercontent.com/58390067/227777859-5e1a498e-1a4a-4431-9846-5be1612b80f5.png)
![image](https://user-images.githubusercontent.com/58390067/227777927-dc689e56-56e3-46bc-b894-281c6e53f262.png)

Can someone help me please?
I don't understand the error

I have chatgpt plus (paid version)
The website works fine
My api key is new I created one for the shortcut

**Comment by Yue-Yang** [comment]
You must add payment method to your api account . Subscription doesn't have relation with api key
